api/v1/service/resource/resource.go

```diff
@@ -183,7 +183,7 @@ func (s *service) ByCatalogKindNameVersion(ctx context.Context, p *resource.ByCa
 func (s *service) ByCatalogKindNameVersionReadme(ctx context.Context,
 	p *resource.ByCatalogKindNameVersionReadmePayload) (*resource.ResourceVersionReadme, error) {
 
-	readmePath := fmt.Sprintf("%s/%s/%s/%s/%s/README.md", s.CatalogClonePath(), strings.ToLower(p.Catalog), strings.ToLower(p.Kind), p.Name, p.Version)
+	readmePath := fmt.Sprintf("%s/%s/%s/%s/%s/README.md", s.CatalogClonePath(), p.Catalog, strings.ToLower(p.Kind), p.Name, p.Version)
```

Comment:
instead of changing here, can't we convert the catalog name to lower case and then store or are we focusing on case sensitive?

Reply:
not sure on this, should we consider catalag name case sensitive? cc: @piyush-garg @PuneetPunamiya